data/src/error_analysis/README.md

```diff
@@ -0,0 +1,45 @@
+# Error Analysis tool for G2P
+
+This tool could be used for performing fine-grain error analysis of a G2P model. It reports a performance matrix of a test output generated by a G2P model. 
+
+Under this error analysis process, each test record generated from a G2P model is evaluated under two criterions. 
+a) if the hypothesized form matches the corpus prediction or not.
+b) if the hypothesized form adheres to grammatical rules of the language.
+
+The output of the tool is a two dimentional matrix as shown below-
+
+```
+               | CG Match  |   CG Not Match |
+---------------|-----------+----------------|
+Pron Match     |  0.7934   |      0.0501    |
+Pron Not Match |  0.1315   |      0.0250    |
+```
+
+Here the numbers represents percentage of test records falling under X section of each category. 
+
+## Prerequisite
+
+### Library
+install pynini 
+
+```
```

Comment:
For code snippets in GitHub-flavored Markdown, right after the initial backticks, you should put the "programming language" included. Here that's just Bash so you add "bash" right after the three ` characters at the beginning (you don't need them at the end).

Reply:
fixed